src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -319,6 +320,7 @@ System.Text.Json.Nodes.JsonValue</PackageDescription>
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
     <Compile Include="System\ReflectionExtensions.cs" />
     <Compile Include="$(CommonPath)System\Obsoletions.cs" Link="Common\System\Obsoletions.cs" />
+    <None Include="readme.md" Pack="true" PackagePath="\"/>
```

Comment:
Nit: shouldn't casing be "Readme.md"?

Reply:
Actually, other markdown files in this repo are cased as "README.md". Should i use that instead?